scIB/integration.py

```diff
@@ -324,22 +324,37 @@ def runConos(adata, batch, hvg=None):
     return out
     """
     
+def runCombat(adata, batch):
+    sc.pp.combat(adata, key=batch)
+    return adata
 
 
-if __name__=="__main__":
-    adata = sc.read('testing.h5ad')
-    #emb, corrected = runScanorama(adata, 'method', False)
-    #print(emb)
-    #print(corrected)
+def runDESC(adata, batch, res=0.8, ncores=24, tmp_dir='/localscratch/'):
```

Comment:
Maybe add the source for the parameters to the docstring. Looks good otherwise

Reply:
good point!

Reply:
883ef0e